peachjam/views/documents.py

```diff
@@ -250,6 +253,36 @@ def get_context_data(self, **kwargs):
         return context
 
 
+@method_decorator(add_slash_to_frbr_uri(), name="setup")
+class SimilarDocumentsView(DetailView):
+    template_name = "peachjam/_similar_documents.html"
+    slug_url_kwarg = "frbr_uri"
+    slug_field = "expression_frbr_uri"
+    model = CoreDocument
```

Comment:
Wrap this view in a permission (perhaps `peachjam_ml.view_documentembedding`) so that we can deploy it and limit access.

Reply:
done